src/index.ts

```diff
@@ -295,7 +316,14 @@ async function main() {
     await searchIndex.createRecommendedIndexes();
   }
 
-  const server = new Marian(searchIndex);
+  const atlasAdmin = new AtlasAdminManager(process.env['ATLAS_ADMIN_PUB_KEY']!, process.env['ATLAS_ADMIN_API_KEY']!);
```

Comment:
I think we should check these env vars instead of doing a non-null assertion to ensure that we have these values populated. We could catch the error here and send a helpful message if these env vars are not set.

Reply:
combined with the group_id logic and added a env var verification